spec/operators/bufferTime-spec.ts

```diff
@@ -88,6 +88,29 @@ describe('bufferTime operator', () => {
     });
   });
 
+  it('should emit buffers at intervals or when the buffer is full, when nexting in subsequent operator', () => {
+    testScheduler.run(({ hot, time, expectObservable, expectSubscriptions }) => {
+      const e1 = hot('  ---a---b---c---d---e---f---g-----|   ');
+      const subs = '    ^--------------------------------!   ';
+      const t = time('  ----------|                          ');
+      const expected = '-------u---v---w---x---y---z-----(c|)';
+      const values = {
+        u: ['a', 'b'],
+        v: ['9', 'c'],
+        w: ['9', 'd'],
+        x: ['9', 'e'],
+        y: ['9', 'f'],
+        z: ['9', 'g'],
+        c: ['9']
+      };
+
+      const result = e1.pipe(bufferTime(t, null, 2, testScheduler), tap(_value => e1.next('9')));
+
+      expectObservable(result).toBe(expected, values);
```

Comment:
As someone working in the codebase for the first time, I "get" marbles & love the idea, but these failure messages are quite obfuscated. I'm sure it's been discussed before, but a more visual output would be nice.

![Screen Shot 2020-06-27 at 11 22 34 AM](https://user-images.githubusercontent.com/4021306/85929908-4104c880-b86d-11ea-811b-3f5329dc87d0.png)